collectors/bzimport/tests/test_convertors.py

```diff
@@ -286,6 +286,7 @@ def test_basic(self):
         assert package_version.version == "version"
         assert package_version.package == package
 
+    @pytest.mark.enable_signals
```

Comment:
I don't know why, but for some reason, this test was preventing the `post_delete` signal to trigger and the flaw_collaborator test was failing, enabling the signals here made it work.

I spent like an hour trying to figure why that happens, but I couldn't